R/imputation.R

```diff
@@ -30,7 +30,7 @@ process_imputation <- function(x, call_attr) {
         conf_level = call_attr[["conf_level"]],
         df = mipool$pooled$df)
     attr(out, "inferences") <- mipool
-    attr(out, "model") <- lapply(mfx_list, attr, "model")
+    attr(out, "model") <- lapply(mfx_list, attr, "model") |> mice::pool()
```

Comment:
Can't use pipe in `R/` code because (I think) that the operator was introduced in 4.1 and we support 3.6. It's fine to keep in the vignettes, though.